### PR TITLE
Function to enable exceptions

### LINKED ISSUE #8: Possibility to enable exceptions
At the moment exceptions are being suppressed and the messages can only be viewed if debug mode is turned on. There should be a way to turn on exceptions without enabling debug mode.

### PR DESCRIPTION
**Pull request by ahmadnazir**
Fixes #8

### CODE REVIEW

File: src/ApiConnector.php [hunk 1]
```diff
@@ -79,6 +85,9 @@ public static function callServer($url, $data=null, $method='get', $options=arra
 			$request = self::$client->createRequest($method, $url, self::$headers, $data, $options);
 			return $request->send();
 		} catch (\Exception $e) {
+            if (self::$throwExceptions) {
+                throw $e;
+            }
```
Comment (janflora) [inline src/ApiConnector.php]: It seems that the indentation has been fucked up here. We need a linter for this repo too..!

Comment (ahmadnazir) [inline src/ApiConnector.php]: I'll fix it.

**Comment by ahmadnazir** [comment]
@janflora Please review again

**Comment by janflora** [comment]
Reviewed and approved.

**Pull request merged**